import BaseException from './BaseException';

class ResourceAlreadyExistsError extends BaseException {
  constructor() {
    super(409, 'Recurso jรก existe');
  }
}

export default ResourceAlreadyExistsError;
